packages/viewer-sandbox/src/main.ts

```diff
@@ -1,42 +1,32 @@
-import { Pane } from 'tweakpane'
-import { Viewer } from '@speckle/viewer'
+import { Viewer, IViewer } from '@speckle/viewer'
 import './style.css'
+import Sandbox from './Sandbox'
 
-const container = document.querySelector<HTMLDivElement>('#renderer')
+const container = document.querySelector<HTMLDivElement>('#renderer') as HTMLElement
 if (!container) {
   throw new Error("Couldn't find #app container!")
 }
 
 // Viewer setup
-const viewer = new Viewer({
-  container,
-  showStats: true
-})
+const viewer = new Viewer(container) as IViewer
```

Comment:
Ditto, don't recast with `as`. If the @speckle/viewer package has real TS types now, then just delete the type augmentations I created for it in the sandbox package.

Reply:
Here the viewer instance is cast to the type which holds the public contract. This was intentional

Reply:
That's not necessary, because Viewer already has a TS type from the class (now that I've fixed it in the local version of the branch, I'm about to push it)

Reply:
It wasn't about necessity, more for verbosity. You want to work with the public contract of the library, not with it's implementation